persistence/sql/persister_consent.go

```diff
@@ -217,6 +217,60 @@ func (p *Persister) GetConsentRequest(ctx context.Context, challenge string) (*f
 	return f.GetConsentRequest(), nil
 }
 
+func (p *Persister) CreateDeviceGrantRequest(ctx context.Context, req *flow.DeviceGrantRequest) error {
```

Comment:
Remove these if you agree we can use `user_code`.

Otherwise, we'll need tests ;).